lib/src/views/splash_view.dart

```diff
@@ -46,65 +47,58 @@ class _SplashViewState extends State<SplashView> with SingleTickerProviderStateM
         child: Stack(
           alignment: Alignment.center,
-          fit: StackFit.expand,
           children: <Widget>[
-            Center(
-              child: AnimatedBuilder(
-                animation: _animationController,
-                builder: (_, __) => SizedBox(
-                  width: 125.0,
-                  child: Stack(
-                    children: <Widget>[
-                      SvgPicture.asset('assets/logo/heart.svg'),
-                      ...[1, 2, 3, 4].map((i) {
-                        var value = _animationController.value * 50;
-                        var y = (value > i * 10 && value < i * 10 + 10) ? value - i * 10 : 0.0;
-                        // Not the animation I wanted but doesn't look too bad lol, so works for now ✌️
-                        return Transform.translate(
-                          offset: Offset(0, -y),
-                          child: SvgPicture.asset('assets/logo/toe$i.svg'),
-                        );
-                      }),
-                    ],
-                  ),
+            AnimatedBuilder(
+              animation: _animationController,
+              builder: (_, __) => SizedBox(
+                width: 125.0,
+                child: Stack(
+                  children: <Widget>[
+                    SvgPicture.asset('assets/logo/heart.svg'),
+                    ...[1, 2, 3, 4].map((i) {
+                      var value = _animationController.value * 50;
+                      var y = (value > i * 10 && value < i * 10 + 10) ? value - i * 10 : 0.0;
+                      // Not the animation I wanted but doesn't look too bad lol, so works for now ✌️
+                      return Transform.translate(
+                        offset: Offset(0, -y),
+                        child: SvgPicture.asset('assets/logo/toe$i.svg'),
+                      );
+                    }),
+                  ],
                 ),
               ),
             ),
-            Positioned(
-              bottom: 40.0,
-              child: Consumer(
-                builder: (_, WidgetRef ref, __) {
-                  return ref.watch(userControllerProvider).when(
-                      data: (User user) {
-                        var nextViewId = (user.status == UserStatus.anonymous || user.status == UserStatus.identified) ? MainView.viewId : AuthenticationView.viewId;
-                        // Future.delayed(Duration.zero) is a bit too hacky to my liking but we need to return a widget and using Navigator otherwise shortcuts the flow, resulting in errors.
-                        Future.delayed(Duration.zero, () => Navigator.pushReplacementNamed(context, nextViewId));
-                        return _buildPlaceholder();
-                      },
-                      error: (Object error, _) {
-                        _animationController.reset();
-                        final isOffline = error is OfflineException;
+            Consumer(
```

Comment:
`Consumer` can be removed since the view became a `ConsumerState`.
You can use the global `ref.watch` 🧞

Reply:
Fixed in latest commit. :)